operator_ui/src/pages/JobsIndex/JobV2Row.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import { TableCell, TableRow, Typography } from '@material-ui/core'
+import { TimeAgo } from 'components/TimeAgo'
+import Link from 'components/Link'
+import { JobSpecV2 } from './JobsIndex'
+import {
+  createStyles,
+  withStyles,
+  WithStyles,
+  Theme,
+} from '@material-ui/core/styles'
+
+const styles = (theme: Theme) =>
+  createStyles({
+    cell: {
+      paddingTop: theme.spacing.unit * 2,
+      paddingBottom: theme.spacing.unit * 2,
+    },
+    link: {
+      '&::before': {
+        content: "''",
+        position: 'absolute',
+        top: 0,
+        left: 0,
+        width: '100%',
+        height: '100%',
+      },
+    },
+  })
+
+interface Props extends WithStyles<typeof styles> {
+  job: JobSpecV2
+}
+
+const JobV2Row = withStyles(styles)(({ job, classes }: Props) => {
+  const createdAt = React.useMemo(() => {
+    const { fluxMonitorSpec, offChainReportingOracleSpec } = job.attributes
+
+    switch (job.attributes.type) {
+      case 'fluxmonitor':
+        return fluxMonitorSpec ? fluxMonitorSpec.createdAt : undefined
```

Comment:
```suggestion
        return fluxMonitorSpec?.createdAt
```

Also, shouldn't need to do `.?` with a type update mentioned above